### PR TITLE
RET-2663 - PDF View link is broken

### PR DESCRIPTION
**Pull request by marcjudge**
### JIRA link
https://tools.hmcts.net/jira/browse/RET-2681

### Change description
Fixing download form not working on claimant applications page.
Also fixes issues with CYA page:

- Hides past employer question when Unfair Dismissal
- Shows notice period question only if it was was asked
- Displays notice period answers when 'Working Notice' is selected

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ikirsanov — DISMISSED** [review]

**Review by gauravhmcts — APPROVED** [review]

File: src/main/routes/routes.ts [hunk 1]
```diff
@@ -174,6 +174,7 @@ export default function (app: Application): void {
   app.get(PageUrls.CLAIM_DETAILS, app.locals.container.cradle.claimDetailsController.get);
   app.get(PageUrls.CITIZEN_HUB_DOCUMENT, app.locals.container.cradle.citizenHubDocumentController.get);
   app.get(PageUrls.GET_CASE_DOCUMENT, app.locals.container.cradle.caseDocumentController.get);
+  app.get(PageUrls.GET_SUMITTED_CASE_DOCUMENT, app.locals.container.cradle.submittedCaseDocumentController.get);
```
Comment (JackReevies) [inline src/main/routes/routes.ts:177]: SUBMITTED?

File: src/main/definitions/constants.ts [hunk 2]
```diff
@@ -156,6 +156,7 @@ export const PageUrls = {
   CITIZEN_HUB_DOCUMENT: '/case-document/:documentType',
   CITIZEN_HUB_DOCUMENT_RESPONSE_RESPONDENT: '/case-document/' + TranslationKeys.CITIZEN_HUB_RESPONSE_FROM_RESPONDENT,
   GET_CASE_DOCUMENT: '/getCaseDocument/:docId',
+  GET_SUMITTED_CASE_DOCUMENT: '/getSubmittedCaseDocument/:docId',
```
Comment (JackReevies) [inline src/main/definitions/constants.ts:159]: SUBMITTED?

**Review by JackReevies — CHANGES_REQUESTED** [review]

File: src/main/routes/routes.ts (hunk 1 above)
Comment (marcjudge) [reply src/main/routes/routes.ts]: Fixed

File: src/main/definitions/constants.ts (hunk 2 above)
Comment (marcjudge) [reply src/main/definitions/constants.ts]: Fixed

**Review by JackReevies — APPROVED** [review]

**Review by Itay-Kirsh — APPROVED** [review]
We should definitely come back to this and try and figure out why a shared document controller didn't work. It's unfortunate to have two separate ones but it's probably best to get this through for now.

**Pull request merged**